cns/fsnotify/fsnotify.go

```diff
@@ -60,13 +60,15 @@ func (w *watcher) releaseAll(ctx context.Context) {
 		file, err := os.Open(filepath)
```

Comment:
The filepath pkg might be a better option here, unsure if CNS runs on both windows/linux